src/app/wallets/ln-send-payment.ts

```diff
@@ -481,6 +491,15 @@ const executePaymentViaLn = async ({
           milliSatsAmount: toMilliSatsFromNumber(amount * 1000),
           maxFee,
         })
+
+    const payment = await lndService.lookupPayment({ pubkey, paymentHash })
+    if (payment instanceof Error) return payment
+
+    payment.paymentRequest = payment.paymentRequest || paymentRequest
+    payment.paymentHash = payment.paymentHash || paymentHash
+    const persistedPayment = await LnPaymentsRepository().update(payment)
+    if (persistedPayment instanceof Error) return persistedPayment
```

Comment:
please move this after settle or void, in this point payment has been made and we should have settle or void as close as possible

Reply:
Same as https://github.com/GaloyMoney/galoy/pull/665#discussion_r746697848